src/TokenHandler/AccountManager.mo

```diff
@@ -128,44 +133,33 @@ module {
 
       if (latestDeposit < prevDeposit) freezeCallback("latestDeposit < prevDeposit on notify");
 
-      let depositInfo = depositRegistry.get(p);
-
       // precredit deposit funds
       if (latestDeposit > fee_) {
-        if (depositInfo.dust == 0) {
+        if (not dust.has(p)) {
```

Comment:
I prefer the condition `prevDeposit > fee_` here.

Reply:
This is the problem we discussed. When the fee changes and the deposit, which was previously less than the fee, becomes more than the fee, such a check is not reliable

Reply:
Yes, but we solved the problem so that the condition now works. If the fee goes down then we recalculate the dust log at the same time we adjust the `fee_` value. A notify() call cannot get in the middle of this.

Reply:
Fixed